zosia_site/database.tf

```diff
@@ -0,0 +1,50 @@
+resource "google_sql_database" "db" {
+  name     = var.db_settings.db_name
+  instance = google_sql_database_instance.db_instance.name
+}
+
+resource "google_sql_database_instance" "db_instance" {
+  name             = var.db_settings.instance_name
+  database_version = var.db_settings.database_version
+
+  settings {
+    tier      = var.db_settings.database_tier
+    edition   = var.db_settings.database_edition
+    disk_size = 10
+
+    backup_configuration {
+      enabled    = true
+      start_time = "02:00"
+    }
+
+    ip_configuration {
+      ipv4_enabled = true
+
+      # TODO: Currently the production zosia database is configured like this 
+      # (no ssl and it accepts connections from every ip)
+      # We should discuss if we want to change this
+      require_ssl = false
+      authorized_networks {
+        value = "0.0.0.0/0"
+      }
+    }
```

Comment:
Do we really need this database to be public? The backend that is running on the app engine can access the database using a private network instead and admins could use some proxy/jumphost if needed.

Reply:
It's a good idea, it will be done in a separate PR.